src/System.Text.Json/src/System/Text/Json/Serialization/ReadStackFrame.cs

```diff
@@ -45,28 +48,28 @@ internal struct ReadStackFrame
         public List<PropertyRef> PropertyRefCache;
 
         /// <summary>
-        /// Is the current object an Enumerable, Dictionary or IDictionaryConstructible.
+        /// Is the current object an Enumerable, IListConstructible, Dictionary or IDictionaryConstructible.
```

Comment:
Is there such a thing as IListConstructible and IDictionaryConstructible?

Seems like the comment should just say "Is the current object and Enumerable or Dictionary"

Reply:
Not anymore. Removed.